pkg/connector/user.go

```diff
@@ -132,7 +135,9 @@ func (o *userResourceType) List(ctx context.Context, parentID *v2.ResourceId, pt
 	for _, user := range users {
 		u, _, err := o.client.Users.GetByID(ctx, user.GetID())
 		if err != nil {
-			return nil, "", nil, err
+			l.Error("error fetching user by id", zap.Error(err), zap.Int64("user_id", user.GetID()))
```

Comment:
should we be checking the status code and only continuing on 404?

Reply:
I wasn't sure. I'll add the check.

Reply:
OK. I tested this by changing the path in Users.GetByID() to always 404. It still fetches stuff, just the emails and a few other details are missing.